doc/source/reference/testing.rst

```diff
@@ -51,6 +51,12 @@ Exceptions and warnings
    errors.OptionError
    errors.OutOfBoundsDatetime
    errors.OutOfBoundsTimedelta
+   errors.PandasChangeWarning
+   errors.Pandas4Warning
+   errors.Pandas5Warning
```

Comment:
Aside from the tests, `Pandas5Warning` isn't used.  Is the idea that we will use it in the future?

Reply:
Yes.